plat/mediatek/mt6795/bl31.ld.S

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
I can see why you've needed to fork the linker script but at some point it would be good to understand if there are any platform hooks we could provide in the generic linker script to help you do what you want without duplicating this whole file.